dev/main.c

```diff
@@ -0,0 +1,146 @@
+#include "main.h"
+#include "Players/players.h"
+#include "Tiles/entity.h"
+//#include "Stages/stage1.h"
+
+unsigned char frame_counter;
+unsigned char volume_atenuation;
+
+Entity* entity;
+
+void loadGraphics2vram(void)
+{
+  /* clear VRAM */
+  SMS_VRAMmemsetW(0, 0x0000, 0x4000);
+
+  // backgound
+  SMS_loadBGPalette(background_palette_bin);
+
+  SMS_loadPSGaidencompressedTiles(backgroundtiles_psgcompr, 0);
+  SMS_loadTileMap(0,0, backgroundtilemap_bin, backgroundtilemap_bin_size);
+  //SMS_setBGPaletteColor(0, RGB(0, 2, 3));
+
+  // items
+  SMS_loadPSGaidencompressedTiles(items_tiles_psgcompr, 18);
+
+  // explosions
+  // SMS_loadPSGaidencompressedTiles(explosions_tiles_psgcompr, 41);
+
+
+  // sprites
+  SMS_loadSpritePalette(spritepalette_bin);
+  SMS_loadTiles(spritetiles_down_bin, PLAYER1_SPRITE_POSITION, 32*6*6);
+  //SMS_setSpritePaletteColor(0, RGB(0, 0, 0));
+
+  // tilelist
+  Tile tilelist1[] = 
+  {
+    // frame 1
+    {11, 0, 0, 0, 0, 1},
```

Comment:
Better to used the designated initializer syntax here - it's more readable, less error-prone (because you don't need to get the order of fields right and can freely change the structure later) and avoids confusing with arrays.

Also, since you're using `stdbool.h` why not use `true` and `false` instead of `0` and `1`?

```suggestion
    { .id = 11, .collidable = false, .flipX = false, .flipY = false, .useSpritePalette = false, .priority = false },
```

Reply:
It was simply more verbose, that's my only reason. I didn't know you can set the parameters like this too and so that's great! 🙂

Reply:
..but very verbose. Actually I intended to make a tool that outputs these lists too.